include/logger/loggerConfig.h

```diff
@@ -357,17 +358,19 @@ typedef struct _GPSConfig {
 #define DEFAULT_GPS_SATELLITE_CONFIG {"GPSSats", "", 0, 20, DEFAULT_GPS_SAMPLE_RATE, 0, 0}
 #define DEFAULT_GPS_QUALITY_CONFIG {"GPSQual", "", 0, 5, DEFAULT_GPS_SAMPLE_RATE, 0, 0}
 #define DEFAULT_GPS_DOP_CONFIG {"GPSDOP", "", 0, 20, DEFAULT_GPS_SAMPLE_RATE, 1, 0}
-
-#define DEFAULT_GPS_CONFIG {                   \
-		DEFAULT_GPS_LATITUDE_CONFIG,           \
-		DEFAULT_GPS_LONGITUDE_CONFIG,          \
-		DEFAULT_GPS_SPEED_CONFIG,              \
-		DEFAULT_GPS_DISTANCE_CONFIG,           \
-		DEFAULT_GPS_ALTITUDE_CONFIG,           \
-		DEFAULT_GPS_SATELLITE_CONFIG,          \
-		DEFAULT_GPS_QUALITY_CONFIG,            \
-		DEFAULT_GPS_DOP_CONFIG                 \
-         }
+#define DEFAULT_GPS_HEADING_CONFIG {"GPSHeading", "Degrees", 0, 360, DEFAULT_GPS_SAMPLE_RATE, 0, 0}
```

Comment:
I think Course is best b/c Track might be confused with the idea of race
tracks, a term we use elsewhere in the system.

On Thu, Jul 30, 2015 at 8:34 AM, Andrew Stiegmann <notifications@github.com>
wrote:

> In include/logger/loggerConfig.h
> <https://github.com/autosportlabs/RaceCapture-Pro_firmware/pull/70#discussion_r35884019>
> :
>
> > @@ -357,17 +358,19 @@ typedef struct _GPSConfig {
> >  #define DEFAULT_GPS_SATELLITE_CONFIG {"GPSSats", "", 0, 20, DEFAULT_GPS_SAMPLE_RATE, 0, 0}
> >  #define DEFAULT_GPS_QUALITY_CONFIG {"GPSQual", "", 0, 5, DEFAULT_GPS_SAMPLE_RATE, 0, 0}
> >  #define DEFAULT_GPS_DOP_CONFIG {"GPSDOP", "", 0, 20, DEFAULT_GPS_SAMPLE_RATE, 1, 0}
> > -
> > -#define DEFAULT_GPS_CONFIG {                   \
> > -		DEFAULT_GPS_LATITUDE_CONFIG,           \
> > -		DEFAULT_GPS_LONGITUDE_CONFIG,          \
> > -		DEFAULT_GPS_SPEED_CONFIG,              \
> > -		DEFAULT_GPS_DISTANCE_CONFIG,           \
> > -		DEFAULT_GPS_ALTITUDE_CONFIG,           \
> > -		DEFAULT_GPS_SATELLITE_CONFIG,          \
> > -		DEFAULT_GPS_QUALITY_CONFIG,            \
> > -		DEFAULT_GPS_DOP_CONFIG                 \
> > -         }
> > +#define DEFAULT_GPS_HEADING_CONFIG {"GPSHeading", "Degrees", 0, 360, DEFAULT_GPS_SAMPLE_RATE, 0, 0}
>
> I think I could get behind "Course". Its similar to track and it seems
> that navigation folks seem to use "Track" and "Course" interchangeably.
> @ryan <https://github.com/ryan>?
> … <#14edf99864428dab_>
> On Thu, Jul 30, 2015 at 10:18 AM, Brent Picasso <notifications@github.com>
> wrote: In include/logger/loggerConfig.h <
> https://github.com/autosportlabs/RaceCapture-Pro_firmware/pull/70#discussion_r35881979>
> : > @@ -357,17 +358,19 @@ typedef struct _GPSConfig { > #define
> DEFAULT_GPS_SATELLITE_CONFIG {"GPSSats", "", 0, 20,
> DEFAULT_GPS_SAMPLE_RATE, 0, 0} > #define DEFAULT_GPS_QUALITY_CONFIG
> {"GPSQual", "", 0, 5, DEFAULT_GPS_SAMPLE_RATE, 0, 0} > #define
> DEFAULT_GPS_DOP_CONFIG {"GPSDOP", "", 0, 20, DEFAULT_GPS_SAMPLE_RATE, 1, 0}
> > - > -#define DEFAULT_GPS_CONFIG { \ > - DEFAULT_GPS_LATITUDE_CONFIG, \ >
> - DEFAULT_GPS_LONGITUDE_CONFIG, \ > - DEFAULT_GPS_SPEED_CONFIG, \ > -
> DEFAULT_GPS_DISTANCE_CONFIG, \ > - DEFAULT_GPS_ALTITUDE_CONFIG, \ > -
> DEFAULT_GPS_SATELLITE_CONFIG, \ > - DEFAULT_GPS_QUALITY_CONFIG, \ > -
> DEFAULT_GPS_DOP_CONFIG \ > - } > +#define DEFAULT_GPS_HEADING_CONFIG
> {"GPSHeading", "Degrees", 0, 360, DEFAULT_GPS_SAMPLE_RATE, 0, 0}
> http://www.americanflyers.net/aviationlibrary/pilots_handbook/images/chapter_14_img_27.jpg
> Course might be best for the actual direction you're travelling. Compass is
> pretty self explanatory. TIL! … <#14edf8b0a6403397_> -Brent On Thu, Jul 30,
> 2015 at 8:14 AM, Ryan Doherty <notifications@github.com> wrote: In
> include/logger/loggerConfig.h <
> https://github.com/autosportlabs/RaceCapture-Pro_firmware/pull/70#discussion_r35881508>
> : > @@ -357,17 +358,19 @@ typedef struct _GPSConfig { > #define
> DEFAULT_GPS_SATELLITE_CONFIG {"GPSSats", "", 0, 20,
> DEFAULT_GPS_SAMPLE_RATE, 0, 0} > #define DEFAULT_GPS_QUALITY_CONFIG
> {"GPSQual", "", 0, 5, DEFAULT_GPS_SAMPLE_RATE, 0, 0} > #define
> DEFAULT_GPS_DOP_CONFIG {"GPSDOP", "", 0, 20, DEFAULT_GPS_SAMPLE_RATE, 1, 0}
> > - > -#define DEFAULT_GPS_CONFIG { \ > - DEFAULT_GPS_LATITUDE_CONFIG, \ >
> - DEFAULT_GPS_LONGITUDE_CONFIG, \ > - DEFAULT_GPS_SPEED_CONFIG, \ > -
> DEFAULT_GPS_DISTANCE_CONFIG, \ > - DEFAULT_GPS_ALTITUDE_CONFIG, \ > -
> DEFAULT_GPS_SATELLITE_CONFIG, \ > - DEFAULT_GPS_QUALITY_CONFIG, \ > -
> DEFAULT_GPS_DOP_CONFIG \ > - } > +#define DEFAULT_GPS_HEADING_CONFIG
> {"GPSHeading", "Degrees", 0, 360, DEFAULT_GPS_SAMPLE_RATE, 0, 0} +1 on
> Compass, not sure about Track, although I can't think of anything better. —
> Reply to this email directly or view it on GitHub <
> https://github.com/autosportlabs/RaceCapture-Pro_firmware/pull/70/files#r35881508>
> . -- Brent Picasso *Autosport Labs <http://www.autosportlabs.com>*
> *Technology for Race and Street* — Reply to this email directly or view it
> on GitHub <
> https://github.com/autosportlabs/RaceCapture-Pro_firmware/pull/70/files#r35881979>
> .
> --
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/autosportlabs/RaceCapture-Pro_firmware/pull/70/files#r35884019>
> .
>



-- 
Brent Picasso
*Autosport Labs <http://www.autosportlabs.com>*
*Technology for Race and Street*